src/cr_kyoushi/statemachines/aecid_attacker/sm.py

```diff
@@ -117,10 +117,24 @@ def build(self, config: StatemachineConfig):
                 escalate_time,
                 name="escalate phase",
             ),
-            name="wait_until_escalate",
+            name="cracking",
             target="cracked_passwords",
         )
 
+        decide_crack_method = NoopTransition(
+            "decide_crack_method",
+            target="crack_choice",
+        )
+
+        crack_wphash = Transition(
+            actions.WPHashCrack(
+                config.wordpress.wl_host,
+                config.wordpress.attacked_user,
+            ),
+            name="crack_wphash",
+            target="reverse_shell_listening",
```

Comment:
The way i understand the flow change here is that you skip the VPN disconnected phase if you do "online" cracking.

If so you should mirror the target of the behavior of the `vpn_reconnect ` transition and  `vpn_reconnected` state.
Meaning you should add another state that copies the behavior of the  `vpn_reconnected` state, but with a different name for the sake of correct logging.

Reply:
I am not sure if I am missing some other inconsistencies in the modified flow. It might be best to add these to the smcat definition so we can get a visual representation making it easier to find sm logic mistakes.

Reply:
I fixed it and updated the smcat.